src/main/java/org/ecocean/ia/WbiaQueueUtil.java

```diff
@@ -0,0 +1,126 @@
+package org.ecocean.ia;
+
+import org.datanucleus.api.rest.orgjson.JSONObject;
+import org.ecocean.cache.CachedQuery;
+import org.ecocean.cache.QueryCache;
+import org.ecocean.cache.QueryCacheFactory;
+import org.ecocean.Util;
+import org.ecocean.RestClient;
+import org.ecocean.identity.IBEISIA;
+import java.net.URL;
+import java.util.Iterator;
+
+public class WbiaQueueUtil {
+
+  //Measurement static values
+  private static JSONObject wbiaQueue = new JSONObject();
+  private static String cacheName="wbiaQueue";
+  
+  private static int numJobs=0;
+  private static int numCompletedJobs = 0;
+  private static int numWorkingJobs = 0;
+  private static int numQueuedJobs = 0;
+  private static int numErrorJobs = 0;
+  private static int numDetectionJobs = 0;
+  private static int numIDJobs = 0;
+  private static int sizeIDJobQueue=0;
+  private static int sizeDetectionJobQueue=0;
+  
+  private static void reloadIfNeeded(boolean refresh) {
+    String context="context0";
+    try {
+      QueryCache qc=QueryCacheFactory.getQueryCache(context);
+      if(qc.getQueryByName(cacheName)!=null && System.currentTimeMillis()<qc.getQueryByName(cacheName).getNextExpirationTimeout() && !refresh){
+        wbiaQueue=Util.toggleJSONObject(qc.getQueryByName(cacheName).getJSONSerializedQueryResult());
+      }
```

Comment:
this `qc.getQueryByName()` seems to be a little bit expensive.  might be better to set a variable to it once, before this, and then just using that variable, rather than calling this potentially three times.   e.g. `queryVal = qc.getQueryByName(cacheName)` and then referencing `queryVal` from there on.